SConstruct

```diff
@@ -748,3 +752,18 @@ if "env" in locals():
     # TODO: replace this with `env.Dump(format="json")`
     # once we start requiring SCons 4.0 as min version.
     methods.dump(env)
+
+
+def print_elapsed_time():
+    elapsed_time_sec = time.time() - time_at_start
+    color_yellow = "\033[93m" if sys.stdout.isatty() else ""
```

Comment:
Color codes won't display correctly on Windows as Python/SCons doesn't set the "enable virtual terminal processing hint", which would allow Windows 10 to interpret the ANSI color codes.

I suggest removing the color codes as this isn't important information anyway, so we don't want it to stand out too much.

Reply:
Ok, I removed the color codes.